internal/k8shandler/forwarding.go

```diff
@@ -313,6 +314,23 @@ func (clusterRequest *ClusterLoggingRequest) verifyInputs(spec *logging.ClusterL
 	}
 }
 
+// LogStoreLokiStackName Name of default lokistack service.
+func (clusterRequest *ClusterLoggingRequest) LogStoreLokiStackName() string {
+	logStore := clusterRequest.Cluster.Spec.LogStore
+	if logStore != nil && logStore.LokiStack.Name != "" {
+		return logStore.LokiStack.Name
+	}
+	return constants.LokiStackName
+}
+
+// DefaultLokiStackURL returns the URL of the default lokistack log store.
+// The tenant and request path will be added.
+func (clusterRequest *ClusterLoggingRequest) DefaultLokiStackURL(tenantPath string) string {
+	name := clusterRequest.LogStoreLokiStackName()
+	path := path.Join("/api/logs/v1", tenantPath)
+	return fmt.Sprintf("https://%s-gateway-http.%s.svc:8080%s", name, constants.OpenshiftNS, path)
```

Comment:
Are we certain the default stack has to be in openshift-logging ?  @xperimental @periklis

Reply:
If not then we need to add a Namespace fileld to LogStore.LokiStack. @xperimental ?

Reply:
There is no mandatory check except docs, but since our Lokistack tenant mode is called out as `openshift-logging` I want to keep our stack for in cluster operations on this namespace only, else folks won't get any logs

Reply:
@periklis does this mean this implementation is acceptable?

Reply:
I think this is the last thread still open on the "old PR": The namespace field has been removed from the LokiStack log-store configuration on purpose. It operates on the assumption that the LokiStack used for logging is in the same namespace as the `ClusterLogging` resource (which will be `openshift-logging` in all cases that I know of).

In general, it is not a big change to add a `Namespace` field to the resource, I just don't know if this is necessary.